### ISSUE TITLE
Clarify good vs poor experience

### ISSUE THREAD

**Issue by rviscomi** [GoogleChrome/web-vitals-extension#42]
CWV defines three buckets of experience: good, needs improvement, and poor. Currently, the extension labels anything not good as poor.

Should the extension account for experiences in the needs improvement bucket?

**Comment by addyosmani** [comment]
Great question, Rick! 

In our initial discussions about badging for the extension, we decided that good vs. poor was sufficient to distinguish if your page needed more work. That said, it would certainly be possible to add a needs improvement bucket into the mix and I'm not opposed :)

It would be a bit of work to add this. Let's keep this open to hear (1) do others want this? (2) if anyone is interested in taking on adding this to the extension.

**Comment by rviscomi** [comment]
Approaching this from a product consistency standpoint, because "poor" and "needs improvement" are loaded terms with threshold/quality implications, I think there's a risk of creating confusion if this tool reports a value like CLS=0.2 as poor while Lighthouse reports it as NI.

**Comment by addyosmani** [comment]
Certainly. Through May we'll be working to land a consistent Core Web Vitals story across Lighthouse, PSI and our other tools. We can consider adding NI here as part of that.

UX: If all metrics are NI, I imagine we'll want to show an orange square and animate the metrics similar to red/poor?

**Comment by balazsorban44** [comment]
I came here for the same request. I am looking at the report of the Chrome Extension on my page:
![image](https://user-images.githubusercontent.com/18369201/93717883-d8ca3800-fb78-11ea-8eda-9e346eacdd29.png)

According to https://web.dev/cls/#what-is-cls, <0.25 should be "needs improvement".

![image](https://web.dev/vitals/cls_8x2.svg)

Though not an issue - it is clear in both cases that further changes are necessary for a good experience - but a bit of consistency would be nice.

A third bucket would also make the transition a bit "softer", not simply two-sided. If am in the middle, I kind of get the feeling that "I am working in the right direction".
